README.md

```diff
@@ -1667,7 +1667,7 @@ This allow you to easily create ACL rules that limit users to a specific Stream
 
 ### Acknowledging Messages
 
-Messages that need acknowledgement will have a Reply subject set, something like `$JS.ACK.ORDERS.test.1.2.2`, this is the prefix defined in `api.JetStreamAckPre` followed by `<stream>.<consumer>.<delivered count>.<stream sequence>.<stream sequence>`.
+Messages that need acknowledgement will have a Reply subject set, something like `$JS.ACK.ORDERS.test.1.2.2`, this is the prefix defined in `api.JetStreamAckPre` followed by `<stream>.<consumer>.<re-delivery count>.<stream sequence>.<delivery sequence>`.
```

Comment:
the `delivery sequence` is `consumer sequence` actually.  Regarding `delivered`, I agree thats a bad choice but its whats used in the code and api outputs etc :(